mis_builder/models/mis_report_instance.py

```diff
@@ -563,6 +563,9 @@ def _compute_pivot_date(self):
     display_columns_description = fields.Boolean(
         help="Display the date range details in the column headers."
     )
+    inverse_detail = fields.Boolean(
+        help="Show detail before KPI.",
```

Comment:
```suggestion
        help="Show account details before KPI.",
```

Also, to nitpick, declare this field next to `no_auto_expand_accounts`.